app/main/views/g_cloud.py

```diff
@@ -129,21 +162,31 @@ def search_services():
     all_frameworks = data_api_client.find_frameworks().get('frameworks')
     framework = framework_helpers.get_latest_live_framework(all_frameworks, 'g-cloud')
 
+    current_lot_slug = get_lot_from_request(request)
+    current_lot = None
+    lots = framework['lots']
+    for lot in lots:
```

Comment:
Impossible, at the moment anyway. There's currently an 'all categories' option on the search page, but that's effectively 'no lot selected' rather than several. (And that's going away for G-Cloud 9 anyway.)